src/app/shared/menu/date/custom-date-range-dropdown.component.ts

```diff
@@ -46,11 +46,15 @@ import { GoogleAnalyticsEventAction, RouteAdvancedRangeAction } from '../../../n
 export class CustomDateRangeDropdownComponent {
   @Input() routerState: RouterModel;
   tempRange: RouterModel;
-  open = false;
   intervalList = IntervalList;
   userChoseRange: string;
   CUSTOM_DATE = 'custom-date';
   STANDARD_DATE = 'standard-date';
+  open = false;
```

Comment:
noticed especially in mobile mode when there is more content to scroll that these dropdowns remained open but were out of place when the window scrolled